pkg/web/api/toproto/processor.go

```diff
@@ -30,7 +30,7 @@ func _() {
 func Processor(in *processor.Instance) *apiv1.Processor {
 	return &apiv1.Processor{
 		Id:        in.ID,
-		Name:      in.Name,
+		Name:      in.Type,
```

Comment:
Note: this field will change once we update to the new proto definition (this PR needs to be merged first so we can update the dependency and then change this field).